src/components/WorkflowEditor.vue

```diff
@@ -900,14 +900,15 @@ export default {
       }
 
       const addCell = ({ title, icon, tooltip = '', width = 60, height = 60, style }) => {
+        const [ttp, label] = this.translateCell(style)
```

Comment:
Lets name it tooltip, also you can remove the tooltip that's passed into it if you can generate it